docs/Course Wiki/ECE Course Offerings/ECE313.md

```diff
@@ -0,0 +1,83 @@
+
+## Prerequisites
+
+- [MATH257](../MATH%20Course%20Offerings/MATH257.md) OR [MATH416](../MATH%20Course%20Offerings/MATH416.md)
+
+The official prerequisite to the course is exactly one of [MATH257](../MATH%20Course%20Offerings/MATH257.md) or [MATH416](../MATH%20Course%20Offerings/MATH416.md). However, neither course is crucial for any content in ECE313. ECE313 lists these as prerequisites because these courses teach students how to deal with difficult mathematical topics - but as an ECE major, students should be getting practice with this every semester anyway. The course is extraordinarily self-contained - much of the prerequisite material required is covered in the Calculus sequence, such as multivariable integrals, summation of geometric series, Taylor series. Everything is built from the axioms, which are taught at the very beginning of the semester.
+
+## When to Take it
+
+It is heavily advised to take this course after [ECE210](ECE210.md), since this course does use convolution. ECE313 will reach convolution before ECE210, hence taking these concurrently will not help either. While ECE313 does fully introduce convolution, the introduction is extremely rushed.
+
+Many computer engineers choose to delay this course until their final semester, as it does not open up many upper-level computer engineering courses. While this may seem like a wise decision in order to avoid a negative impact on your GPA, be warned that the likelihood of failing this class is not zero, especially during your final semester when you are mentally checked out.
+
+Electrical engineers are advised to take this course the semester after they take ECE210, as many upper-level electrical engineering courses in areas such as signal processing, communications, controls, and remote sensing require ECE313 as a prerequisite (and for a good reason). This course is also extremely important for those interested in theoretical computer science or machine learning.
+
+## Course Structure
+
+During the semester, the course assigns 12-13 written homeworks, due every week, and 12 PrairieLearn homeworks (with infinite tries), due every week. There are two midterms and one final.
+
+The course starts off extremely light, as the axioms of probability are often facts that students have known through prior experience. However, it will continually ramp up throughout the semester, with each topic building upon the previous one. Do not let this course blindside you - it can be extremely dangerous for your grade.
```

Comment:
The course starts off extremely light -> Many students find the start of ECE313 light, since the axioms of probability are often facts they've seen before. However,...